### DISCUSSION TITLE
Offer an option to delete personal data of past events

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2370] [Ideas]
Hi,

since some events or organizers are not legally obliged to keep record of past orders (cash payments at the venue, using pretix for capacity planning for example), it would be very nice to give them an option to delete all collected personal data when the event is over. We would prefer to only store as much personal data as necessary, so shredding everything connected to past events would be preferred.

Thanks,
Alex

**Comment by raphaelm** [comment]
It's absolutely possible to shred info of a past event, you can find it via the same menu as event deletion.

**Reply by choseh** [reply]
sorry, didn't see this at first. question is, will that work for subevents? looks like there's no dangerzone for subevents. so we just want to delete all customer data for one of the dates in that event series (and others in the future)

**Comment by choseh** [comment]
@raphaelm Will you add customer data shredding for subevents in the future, or will this be exclusive to the Corona Shredder plugin? In case of plugin functionality, should I contact you guys via support@ to discuss pricing? Thanks
